src/Orchestration/Orchestration.php

```diff
@@ -19,6 +20,59 @@ public function __construct(Adapter $adapter)
         $this->adapter = $adapter;
     }
 
+    /**
+     * Command Line String into Array
+     *
+     * @param string $command
+     *
+     * @return (false|string)[]
+     *
+     * @psalm-return list<false|string>
```

Comment:
Why do we define the return type twice with `@return` and `@psalm-return`?

Reply:
psalm itself added that when I ran it with --alter to find and fix issues